frontend/js/lib/vanillaSelectBox.js

```diff
@@ -0,0 +1,675 @@
+/*
```

Comment:
For 3rd party files, it's best to put them all in one folder called third_party and be sure to add any necessary licenses.

Reply:
a new `third_party folder` is created which contains all third_party files. Paths in index.html file are modified accordingly